%YAML:1.0
---
start_point : [ -5.2092399999999997e-02, 3.4790799999999997e-02,
    -2.4697600000000000e+00 ]
loading_point : [ 2.0173399999999999e+00, -3.4642100000000000e+00,
    -8.6299300000000002e+01 ]
unloading_point : [ -2.4026999999999998e+00, -5.6779799999999998e+00,
    8.8649699999999996e+01 ]
first_rest_point : [ 1.8246000000000000e+00, -1.0750299999999999e+00,
    -5.7899700000000003e+01 ]
second_rest_point : [ 8.5314199999999996e-01, -4.2294600000000004e+00,
    9.2422200000000004e+01 ]
